packages/sources/enzyme/README.md

```diff
@@ -0,0 +1,96 @@
+# Chainlink External Adapter for Enzyme
+
+Adapter to interact with Enzyme contracts.
+
+### Environment Variables
+
+| Required? |  Name   |                                Description                                 | Options | Defaults to |
+| :-------: | :-----: | :------------------------------------------------------------------------: | :-----: | :---------: |
+|    ✅     | RPC_URL | An http(s) RPC URL to a blockchain node that can read the Enzyme contracts |         |             |
+
+---
+
+### Input Parameters
```

Comment:
Can be `##`

Reply:
This is from our template, we can leave as-is.